internal/users/postgresrepo.go

```diff
@@ -0,0 +1,71 @@
+package users
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	selectByID = `
+SELECT id, name, role, email, password
+FROM "USER"
+WHERE id = $1`
+
+	selectByCreds = `
+SELECT id, name, role, email, password
+FROM "USER"
+WHERE email = $1 AND password = $2`
+)
+
+// PostgresRepository is implementation of
+// UserRepository for postgres database.
+type PostgresRepository struct {
```

Comment:
Don't we use MySQL instead of that? I think that we use a driver (Bridge pattern https://en.wikipedia.org/wiki/Bridge_pattern). It's not a good name at all

Reply:
Also, try read a little more about Repository https://martinfowler.com/eaaCatalog/repository.html